models/ncNewsModels.js

```diff
@@ -8,8 +8,13 @@ exports.fetchAllTopics = () => {
 };
 exports.fetchArticleById = (article_id) => {
     const queryParams = [article_id];
-    const query = `SELECT * FROM articles
-    WHERE article_id = $1`;
+    const query = `SELECT articles.*,
+    COUNT(comments.article_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    WHERE articles.article_id = $1
+    GROUP BY articles.article_id
+    `;
```

Comment:
Nice work with the query, you can additionally look into casting the data type to be a number rather than the default string - worth having a look into!

Reply:
I wondered if I could do that. I'll have a look.